File: CMS/src/app/shared/DanhMucSanPhamPhanLoai.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucSanPhamPhanLoai } from 'src/app/shared/DanhMucSanPhamPhanLoai.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucSanPhamPhanLoaiService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucSanPhamPhanLoai";
    }
}

File: Service/Implement/DanhMucSanPhamPhanLoaiService.cs
namespace Service.Implement
{
    public class DanhMucSanPhamPhanLoaiService : BaseService<DanhMucSanPhamPhanLoai, IDanhMucSanPhamPhanLoaiRepository>
    , IDanhMucSanPhamPhanLoaiService
    {
    private readonly IDanhMucSanPhamPhanLoaiRepository _DanhMucSanPhamPhanLoaiRepository;
    public DanhMucSanPhamPhanLoaiService(IDanhMucSanPhamPhanLoaiRepository DanhMucSanPhamPhanLoaiRepository) : base(DanhMucSanPhamPhanLoaiRepository)
    {
    _DanhMucSanPhamPhanLoaiRepository = DanhMucSanPhamPhanLoaiRepository;
    }
    }
    }

